template-only-bin/set-up-project.sh

```diff
@@ -34,3 +34,6 @@ sed -i.bak "s|<REPO_URL>|$REPO_URL|" main.tf
 # Replace remaining placeholder values
 sed -i.bak "s/<OWNER>/$OWNER/" main.tf
 sed -i.bak "s/<DEFAULT_REGION>/$DEFAULT_REGION/" main.tf
+
+# Remove the backup file created by sed
+rm main.tf.bak
```

Comment:
It's because `sed` behaves differently on Mac and github actions due to different versions of sed being used. On one of the versions, the `-i` flag is required in order to edit files in place. Without `-i` the files always end up blank.

Should I document that as a comment? Not sure where to put that comment. Does it make sense to put it here, or up above where we pass in the `-i` flag?